mobile/src/navigation/app/components/drawer-item/drawer-item.tsx

```diff
@@ -22,7 +21,12 @@ const DrawerItem: FC<DrawerNavigationItem> = ({ name, isFocused = false }) => {
       style={{ backgroundColor, ...styles.item }}
       onPress={handlePress}
     >
-      <Text style={{ color, ...styles.label }}>{name}</Text>
+      <View style={styles.iconContainer}>
+        <Icon name={name} color={color} width={20} height={20} />
```

Comment:
You need to add another property to DrawerNavigationItem - icon: IconName, don't use name property for this
```suggestion
        <Text style={{ color, ...styles.label }}>{icon}</Text>
```